api/RTCPeerConnection.json

```diff
@@ -1506,6 +1508,7 @@
       "getRemoteStreams": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/getRemoteStreams",
+          "spec_url": "https://w3c.github.io/webrtc-pc/#dom-peerconnection-getreceivers",
```

Comment:
This isn't matching

Reply:
`getRemoteStreams` is deprecated and standard_track:false, and not a member of RTCPeerConnection in the spec, and https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/getRemoteStreams redirects to https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/getReceivers, so I’ve updated the patch here to drop the `spec_url` here.